tests/e2e_tests/cypress/support/commands.js

```diff
@@ -0,0 +1,25 @@
+// ***********************************************
+// This example commands.js shows you how to
+// create various custom commands and overwrite
+// existing commands.
+//
+// For more comprehensive examples of custom
+// commands please read more here:
+// https://on.cypress.io/custom-commands
```

Comment:
Może tę jedną linijkę, lub całą treść przenieść gdzieś do readme / wiki, a resztę pliku usunąć? W zasadzie to jest jeden duży komentarz do JS-a, co jako JS sensu nie ma, a jako dokumentacja jest mało czytelne.

Reply:
Zupełnie się z Tobą zgadzam... niestety nie zgadza się ze mną cypress, który wymaga tego pliku :(

Jeśli jest jakaś opcja na to, to z wielką chęcią wywalę :)